engine/runtime-parser/src/main/java/org/enso/compiler/core/TreeToIr.java

```diff
@@ -37,7 +38,7 @@ private TreeToIr() {
     * @param ast the tree representing the program to translate
     * @return the IR representation of `inputAST`
     */
-  IR.Module translate(Tree ast) {
+  org.enso.compiler.core.ir.Module translate(Tree ast) {
```

Comment:
With explicit `import org.enso.compiler.core.ir.Module` it should be safe to use just `Module`. It will take precedence over `java.lang.Module`.